docs/components/board/_index.md

```diff
@@ -546,6 +546,71 @@ err := myBoard.WriteAnalog(context.Background(), "11", 48, nil)
 {{% /tab %}}
 {{< /tabs >}}
 
+### StreamTicks
+
+Start a stream of [`DigitalInterrupt`](/components/board/#digital_interrupts) ticks.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `interrupts` (List[[DigitalInterrupt](https://python.viam.dev/autoapi/viam/components/board/board/index.html#viam.components.board.board.Board.DigitalInterrupt)]): List of digital interrupts to receive ticks from.
+
+**Returns:**
+
+- [(`TickStream`)](https://python.viam.dev/autoapi/viam/components/board/board/index.html#viam.components.board.board.TickStream): [Stream](https://python.viam.dev/autoapi/viam/streams/index.html#viam.streams.Stream) of [Ticks](https://python.viam.dev/autoapi/viam/proto/component/board/index.html#viam.proto.component.board.StreamTicksResponse), objects containing `pin_name`, `time`, and `high` fields.
+
+```python {class="line-numbers linkable-line-numbers"}
+my_board = Board.from_robot(robot=robot, name="my_board")
+
+di8 = await my_board.digital_interrupt_by_name(name="8")
+di11 = await my_board.digital_interrupt_by_name(name="11")
+
+# Stream ticks from the listed digital interrupts on pins 8 and 11.
+ticks = await my_board.stream_ticks([di8, di11])
```

Comment:
@oliviamiller noticed code sample doesn't use await but this is async-- is that because of stream or would it normally use await? can remove 

```suggestion
ticks = my_board.stream_ticks([di8, di11])
```